lib/httplib/connstate.go

```diff
@@ -0,0 +1,237 @@
+/*
+Copyright 2020 Gravitational, Inc.
```

Comment:
```suggestion
Copyright 2021 Gravitational, Inc.
```

Reply:
oof. I actually copied this from one of the other files.

Reply:
`make fix-license` FTW :-)